src/operation/emulation_model.jl

```diff
@@ -221,7 +221,9 @@ function model_store_params_init!(model::EmulationModel)
 end
 
 function model_store_init!(model::EmulationModel)
-    initialize_model_storage!(
+    # TODO DT: style of function names: verb_noun or noun_verb?
```

Comment:
Potential naming consistency issue

Reply:
I have no preference, but yeah needs to be made consistent

Reply:
I will make it verb_noun